salvador/ViewModels/GenerateImageViewModel.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ContentView-ViewModal.swift
+//  salvador
+//
+//  Created by Geury Roustand on 13.02.23.
+//
+
+import Foundation
+import MapKit
+import SwiftUI
+
+
+
+
+//extension GenerateImageView  {
```

Comment:
Yes the reason I commented out because when I was trying to call the model on the test file, I was not able to find the GenerateImageViewModel  function,  what I understand is that the extension makes the modal somehow private so that I can only call it in the GenerateImageView 


Please correct me if I'm wrong and if there is another way to do this

Reply:
If you go with the extension, you need to use the correct namespace when you use it e.g.:
```
@MainActor class UnitTestingGenerateImageViewModel_Tests: XCTestCase {
    
    var modelView : GenerateImageView.GenerateImageViewModel!

    @MainActor override func setUpWithError() throws {
        modelView = GenerateImageView.GenerateImageViewModel()
```
That is also the reason why I would name it just `ViewModel` in case you go with the extension.
It will shorten the naming to `GenerateImageView.ViewModel`

Reply:
Yes that make more sences , thank u!!